test/integration/suite_test.go

```diff
@@ -143,6 +143,11 @@ func TestMain(m *testing.M) {
 				exitOnErr(err)
 			}
 		}
+		fmt.Println("INFO: configuring feature gates")
+		if controllerFeatureGates == "" {
+			controllerFeatureGates = "Gateway=true"
```

Comment:
Perhaps place `"Gateway=true"` in a named `const` with a comment?

Reply:
Sounds good! added in db3d5f9a376e55ad53aeb9db7f98261d59ae1f6f